From 69858aa741c658dfeab6a08e5b196ecda2571b0c Mon Sep 17 00:00:00 2001
From: Emmanuel Berthier <emmanuel.berthier@intel.com>
Date: Fri, 26 Jun 2015 19:31:02 +0200
Subject: [PATCH] EM/TYPE-C:AddressSanitizer: out of bounds access to
 fusb300_acpi_match

This patch is to fix the buffer overflow in fusb300_acpi_match
table which is reported by the addressSanitizer.

Change-Id: I2d2e560d2dff4347f0596e9dcd9accda88cb15ff
Tracked-On: https://jira01.devtools.intel.com/browse/IMINAN-40467
Signed-off-by: Emmanuel Berthier <emmanuel.berthier@intel.com>
Signed-off-by: Biju Chakkachattil <bijux.chakkachattil@intel.com>
---
 drivers/usb/typec/phy-fusb300.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/usb/typec/phy-fusb300.c b/drivers/usb/typec/phy-fusb300.c
index 50e2340db674..9c5bc0d8f607 100644
--- a/drivers/usb/typec/phy-fusb300.c
+++ b/drivers/usb/typec/phy-fusb300.c
@@ -1616,6 +1616,7 @@ static const struct dev_pm_ops fusb300_pm_ops = {
 #ifdef CONFIG_ACPI
 static struct acpi_device_id fusb300_acpi_match[] = {
 	{"FUSB0300", 0},
+	{ }
 };
 MODULE_DEVICE_TABLE(acpi, fusb300_acpi_match);
 #endif
-- 
1.9.1

